docs/release_notes/2024/sept2024.md

```diff
@@ -9,6 +9,27 @@ tags:
   - september
 ---
 
+## 3.3.11.\* (September 20, 2024)
+
+- Prophecy Python libs version: 1.9.14
+- Prophecy Scala libs version: 8.1.4
+
+### Features {#Features3311}
+
+#### Spark Copilot Enhancements {#Spark3311}
+
+- **Config syntax**: We've unified the config syntax across all project types by supporting Jinja config syntax. This means that you can use `{{foo}}` for variable substitution.
+
+#### SQL Copilot Enhancements {#SQL3311}
+
+- **Lineage run**: We now support SQL run history and grouping for SQL runtime logs. We also allow you to diagnose seed error performance.
+
+- **Parametrized schema**: In some scenarios, you may need to parameterize your location paths using external variables. You can do this by enabling the Schema field parameterization on the Location tab of your Target Model.
+
+### Minor Improvements {#MinorImprovements3311}
+
+- **Deployment APIs**: You can use our external APIs for deployment and monitoring of your Prophecy project.
```

Comment:
Link to API docs when we have them

Reply:
@pankajb23 is there a link to the APIs that we can share?

Reply:
Waiting on respone from Pankaj and rest of the team here and on https://app.asana.com/0/1206327447905631/1208168278656704/f. Also on Slack.

Reply:
Did not receive any response.